client/src/components/AboutGalaxy.vue

```diff
@@ -0,0 +1,72 @@
+<script setup>
+/* global __buildTimestamp__, __license__  */
+/* (injected by webpack) */
+
+import { computed } from "vue";
+
+import { getAppRoot } from "onload/loadConfig";
+import { useConfig } from "composables/useConfig";
+import UtcDate from "components/UtcDate";
+import License from "components/License/License";
+import ExternalLink from "components/ExternalLink";
+
+const { config, isLoaded } = useConfig();
+
+const clientBuildDate = __buildTimestamp__ || new Date();
+const apiDocsLink = `${getAppRoot()}api/docs`;
+const galaxyLicense = __license__;
+
+const versionUserDocumentationUrl = computed(() => {
+    const configVal = config.value;
+    return config.value.version_minor.slice(0, 3) == "dev"
+        ? "https://docs.galaxyproject.org/en/latest/releases/index.html"
+        : `${configVal.release_doc_base_url}${configVal.version_major}/releases/${configVal.version_major}_announce_user.html`;
+});
+
+const versionExtra = computed(() => {
+    const configVal = config.value;
+    return Object.entries(configVal.version_extra);
+});
+</script>
+
+<template>
+    <div v-if="isLoaded">
+        <h1>About This Galaxy</h1>
+        <div>
+            <!-- Galaxy version (detailed), with a link to the release notes -->
+            <h4>Galaxy Version Information</h4>
+            <p>
+                The Galaxy Server is running version
+                <external-link :href="versionUserDocumentationUrl">
+                    <strong> {{ config.version_major }}.{{ config.version_minor }}</strong> </external-link
+                >, and the web client was built on <UtcDate :date="clientBuildDate" mode="pretty" />.
+            </p>
+            <template v-if="config.version_extra">
+                <p>The server also provides the following extra version information</p>
+                <b-table :striped="true" thead-class="d-none" :items="versionExtra" />
```

Comment:
Small variant is more in-line with the style of the rest of the page.

```suggestion
                <b-table striped small thead-class="d-none" :items="versionExtra" />
```

Not too important, but the table still differentiates itself from the background rather poorly, especially if there is an even number of entries in the `version.json`. I feel like it should have an outline, but the built in `outlined` won't work here, as it relies on a header being present.
We could introduce our own css, or wrap it in some form of outline, maybe a `b-card`?

Reply:
Most instances won't even have this, and will have one or two rows max, so a visually larger component didn't make much sense here.  Maybe I chuck the table and just list any keys; I'll give that a shot.